tests/CMakeLists.txt

```diff
@@ -149,16 +149,16 @@ ConfigureTest(TRACKING_TEST mr/device/tracking_mr_tests.cpp GPUS 1 PERCENT 100)
 ConfigureTest(FAILURE_CALLBACK_TEST mr/device/failure_callback_mr_tests.cpp)
 
 # prefetch adaptor tests
-ConfigureTest(PREFETCH_ADAPTOR_TEST mr/device/prefetch_resource_adaptor_tests.cpp)
+ConfigureTest(PREFETCH_ADAPTOR_TEST mr/device/prefetch_resource_adaptor_tests.cpp GPUS 1 PERCENT 100)
```

Comment:
Were these cmake changes accidentally added? I don't think these should be in the PR.

Reply:
We've had a lot of flaky test failures in RMM lately due to parallel test execution causing OOM errors. It was advised in an offline discussion to try and set some of the failing tests to use `PERCENT 100` to avoid OOM-ing.

Reply:
@harrism @kingcrimsontianyu If we don't reach consensus on doing this here, let's consider separating these changes into another PR. We do need a fix for the flaky CI failures from OOM errors.

Reply:
Got it.

Reply:
Oh, I already submitted the merge command since we have approvals.

Reply:
Hmmm, `prefetch_resource_adaptor_tests.cpp` allocates at most 400KB from what I can tell.

`limiting_mr_tests.cpp` allocates 10s of MB. So also not much.

`mr/host/mr_ref_tests.cpp` allocates HOST memory at most 1GiB per test.

Why were these tests chosen to change?

Reply:
As suggested, I have reverted the CMake script changes for this MR. 
PS: Also manually squashed the commits into a single signed one.